static/src/javascripts/projects/common/modules/commercial/third-party-tags/facebook.js

```diff
@@ -0,0 +1,87 @@
+/* global FB */
+/* eslint no-console:0 */
+define([
+    'fastdom',
+    'common/utils/detect',
+    'common/utils/cookies',
+    'common/utils/template',
+    'text!common/views/commercial/facebook.html'
+], function(fastdom, detect, cookies, template, facebookStr) {
+    var scriptId = 'facebook-jssdk';
+    var scriptSrc = '//connect.facebook.net/en_US/sdk/xfbml.ad.js#xfbml=1&version=v2.5';
+    var appNetworkId = '978824118882656';
+    var adUnits = [
+        {
+            referenceNode: '#top > .l-side-margins > :first-child',
+            where: 'beforebegin',
+            placementId: '978824118882656_978826235549111',
+            adId: 'ad_root'
+        }
+    ];
+    var facebookTpl;
+
+    return {
+        load: init
```

Comment:
I _think_ I can see why we've named this method this way - `load` has more meaning for consumers of the module; `init` has more meaning inside the module - but couldn't we expose just a singular `loadFacebook` function?